src/serialize/xml/normalize.ts

```diff
@@ -564,7 +565,7 @@ export class LicenseNormalizer extends BaseXmlNormalizer<Models.License> {
           ? undefined
           : this._factory.makeForAttachment().normalize(data.text, options, 'text'),
         XmlSchema.isAnyURI(url)
-          ? makeTextElement(url, 'url')
+          ? makeTextElement(percentEncodeUrl(url), 'url')
```

Comment:
`percentEncodeUrl()`  should be applied before `XmlSchema.isAnyURI(url)` is checked